.mise.toml

```diff
@@ -0,0 +1,3 @@
+[tools]
+node = "lts-iron"
+yarn = "4"
```

Comment:
yarn should no longer be (globally) installed, and we should use corepack, the version of yarn is defined in `package.json` `packageManager`. Can we use that instead?

Reply:
good point. Then we can get rid of .mise.toml, corepack is there without the setting: MISE_NODE_COREPACK